duckchat/duckchat-impl/src/main/java/com/duckduckgo/duckchat/impl/ui/DuckChatOmnibarLayout.kt

```diff
@@ -220,13 +220,13 @@ class DuckChatOmnibarLayout @JvmOverloads constructor(
         }
     }
 
-    fun submitMessage() {
-        val message = duckChatInput.text.toString().trim()
-        if (message.isNotEmpty()) {
+    fun submitMessage(message: String? = null) {
+        val text = message?.also(duckChatInput::setText) ?: duckChatInput.text.toString().trim()
+        if (text.isNotEmpty()) {
```

Comment:
Maybe we want to check is not blank instead?
```suggestion
        if (text.isNotBlank()) {
```